### PR TITLE
adding new error codes for extend action to reissue api

### PR DESCRIPTION
**Pull request by lalottacontinua**

### CODE REVIEW

File: dcc-reissue-api.yaml [hunk 1]
```diff
@@ -183,6 +183,26 @@ paths:
...
+                  value:
+                    error: RI400-1600
+                    message: expiration date too old to become extended
+                TooEarlyExpirationDate:
+                  description: The expiration date of the last issued cert is too early.
+                  value:
+                    error: RI400-1700
+                    message: expiration date too young to become extended
+                SameIssuanceDate:
+                  description: Two or more certificates are issued at the same date.
```
Comment (MikeMcC399) [inline dcc-reissue-api.yaml:197]: "at the same date"
=> "on the same date"

The preposition for date is "on".

File: dcc-reissue-api.yaml [hunk 2]
```diff
@@ -183,6 +183,26 @@ paths:
...
+                  value:
+                    error: RI400-1800
+                    message: at least two DCCs were issued the same date
```
Comment (MikeMcC399) [inline dcc-reissue-api.yaml:200]: "issued the same date" 
maybe better "issued for the same date" ?

File: dcc-reissue-api.yaml [hunk 3]
```diff
@@ -183,6 +183,26 @@ paths:
...
+                TooOldExpirationDate:
+                  description: The expiration date of the last issued cert is too old.
+                  value:
+                    error: RI400-1600
+                    message: expiration date too old to become extended
+                TooEarlyExpirationDate:
+                  description: The expiration date of the last issued cert is too early.
+                  value:
+                    error: RI400-1700
+                    message: expiration date too young to become extended
```
Comment (MikeMcC399) [inline dcc-reissue-api.yaml:195]: "too young" => "too recent"

for dates, a better opposite of "old" is "recent"

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Some grammar comments added regarding dates.

File: dcc-reissue-api.yaml (hunk 1 above)
Comment (lalottacontinua) [reply dcc-reissue-api.yaml]: @MikeMcC399 sorry, my last changes weren't included. I hope, it's more clear now.

File: dcc-reissue-api.yaml (hunk 3 above)
Comment (lalottacontinua) [reply dcc-reissue-api.yaml:195]: @MikeMcC399 sorry, my last changes weren't included. I hope, it's more clear now.

File: dcc-reissue-api.yaml (hunk 2 above)
Comment (lalottacontinua) [reply dcc-reissue-api.yaml]: @MikeMcC399 sorry, my last changes weren't included. I hope, it's more clear now.

File: dcc-reissue-api.yaml (hunk 3 above)
Comment (lalottacontinua) [reply dcc-reissue-api.yaml:195]: @MikeMcC399 I do agree. The message included is though already deployed on our QA stage. Should I fix it as a hotfix? Or should we leave it like it is for now and then fix it in the next release?

Comment (MikeMcC399) [reply dcc-reissue-api.yaml:195]: @lalottacontinua 
It's not critical. "young" is still understandable, even if it is not common usage in this context.

Comment (lalottacontinua) [reply dcc-reissue-api.yaml:195]: Okay, thanks. I will take it into account for our next release.

**Pull request merged**